zeppelin-server/src/main/java/org/apache/zeppelin/rest/InterpreterRestApi.java

```diff
@@ -56,18 +63,32 @@
 @Path("/interpreter")
 @Produces("application/json")
 public class InterpreterRestApi {
+
   private static final Logger logger = LoggerFactory.getLogger(InterpreterRestApi.class);
 
-  private InterpreterSettingManager interpreterSettingManager;
-  private NotebookServer notebookServer;
+  private final InterpreterService interpreterService;
+  private final InterpreterSettingManager interpreterSettingManager;
+  private final NotebookServer notebookServer;
+  private final Gson gson;
 
   public InterpreterRestApi() {
+    //TODO(jl): Is this constructor needed?
```

Comment:
If it is not used, then I think we can remove it.

Reply:
Fixed